schema/snapcraft.json

```diff
@@ -914,6 +914,8 @@
                                     "gnome-3-34",
                                     "gnome-3-38",
                                     "kde-neon",
+                                    "qt5-15",
+				    "qt6-5",
```

Comment:
It doesn't appear in github, but this line is all tabs.  Can you convert them to spaces?